packages/babel/cli.js

```diff
@@ -1,10 +1,13 @@
 #!/usr/bin/env node
+const isGlobal = require("is-global");
+
+const globalMessage = isGlobal() ? " -g" : "";
 
 console.error("You have mistakenly installed the `babel` package, which is a no-op in Babel 6.\n" +
   "Babel's CLI commands have been moved from the `babel` package to the `babel-cli` package.\n" +
   "\n" +
-  "    npm uninstall babel\n" +
-  "    npm install babel-cli\n" +
+  "    npm uninstall" + globalMessage + " babel-cli\n" +
```

Comment:
Ah, I see. Other than changing `babel-cli` back to `babel` is there any other feedback for this PR? Also is it okay just to add another commit to make this change or do you prefer I rebase to keep the commit history clean?